pkg/operator/operator.go

```diff
@@ -302,12 +306,17 @@ func (c *csiDriverOperator) handleSync(instance *v1alpha1.EBSCSIDriver) error {
 		return fmt.Errorf("failed to sync DaemonSet: %v", err)
 	}
 
+	credentialsRequest, err := c.syncCredentialsRequest(instance)
+	if err != nil {
+		return fmt.Errorf("failed to sync CredentialsRequest: %v", err)
+	}
+
```

Comment:
I'd preferably create this resource earlier

Reply:
Moved right after Namespace creation